includes/wikia/GlobalFunctions.php

```diff
@@ -1557,3 +1557,17 @@ function wfGetBaseDomainForHost( $host ) {
 
 	return $wgWikiaBaseDomain;
 }
+
+function wfDoesURLHaveFandomDomain( $url ): bool {
+	global $wgFandomBaseDomain;
+
+	// e.g. not existing wikis
+	if ( empty( $url ) ) {
+		return false;
+	}
+
+	$host = parse_url( $url, PHP_URL_HOST );
```

Comment:
`$host` can be `false`

Reply:
I moved the check in https://github.com/Wikia/app/pull/16076/commits/6a0e895d0637680f6827608e26306b4ecb5d6e01